manim/cli/render/render_options.py

```diff
@@ -59,15 +61,20 @@ def validate_resolution(ctx, param, value):
         "-q",
         "--quality",
         default=None,
-        type=click.Choice(["l", "m", "h", "p", "k"], case_sensitive=False),
-        help="""
-            Render quality at the follow resolution framerates, respectively:
-            854x480 30FPS,
-            1280x720 30FPS,
-            1920x1080 60FPS,
-            2560x1440 60FPS,
-            3840x2160 60FPS
-            """,
+        type=click.Choice(
+            reversed([q["flag"] for q in QUALITIES.values() if q["flag"]]),
```

Comment:
Curiously, this line caused the `%%manim` IPython magic command to break on the second command. It needs to be wrapped in `list()`, then it works properly. I'll prepare a bugfix PR and do a bugfix release.